framework/include/media/MediaRecorder.h

```diff
@@ -165,6 +178,18 @@ class MediaRecorder
 	 */
 	recorder_result_t setObserver(std::shared_ptr<MediaRecorderObserverInterface> observer);
 
+	/**
+	 * @brief Set limitation of recording time by given value(second), will be stopped when it reaches that.
+	 *          This shoud be called after create but before prepare
```

Comment:
Sentence started after a tab, may be re-align to beginning ?

Reply:
typo: Shoud => should

Reply:
fixed.